mk/compile.mk

```diff
@@ -33,3 +33,7 @@ run: pbapi ## Build and run backend API
 .PHONY: clean
 clean: ## Clean build artifacts
 	rm pbapi pbmigrate pbworker
+
+.PHONY: generate
+generate: ## Run go generate tool (stringer etc)
+	go generate ./...
```

Comment:
What is the goal of the target?

Reply:
A leftover from my experiments, removing.